### PR TITLE
Actions.auth rework for pipeline

### PR DESCRIPTION
**Pull request by uhriap**

### CODE REVIEW

File: cocaine/tools/actions/auth.py [hunk 1]
```diff
@@ -8,124 +8,141 @@
...
 
         raise gen.Return({
             'token': token,
             'members': members,
         })
 
+    @coroutine
+    def list_groups(self):
+        channel = yield self._storage.find(COLLECTION_GROUPS, ['auth'])
+        yield channel.rg.get()
```
Comment (3Hren) [inline cocaine/tools/actions/auth.py]: Typo: `rg` -> `rx`.

Comment (3Hren) [inline cocaine/tools/actions/auth.py]: Moreover it must return something instead of `None`.

File: cocaine/tools/actions/auth.py [hunk 2]
```diff
@@ -8,124 +8,141 @@
...
-        self._name = name
-        self._token = token
-        self._force = force
-        self._storage = storage
+        self._storage = storage  # HINT: storage.write(namespace, key, value, tags)
 
     @coroutine
-    def execute(self):
-        if not self._force:
+    def create_group(self, group_name, token, force=False):
```
Comment (3Hren) [inline cocaine/tools/actions/auth.py]: You can't just rename arguments without synchronizing with https://github.com/cocaine/cocaine-tools/blob/master/cocaine/tools/dispatch.py.

Comment (uhriap) [inline cocaine/tools/actions/auth.py]: names reverted

File: cocaine/tools/actions/auth.py (hunk 1 above)
Comment (uhriap) [inline cocaine/tools/actions/auth.py]: fixed

**Comment by 3Hren** [comment]
```
esafronov@213:~/code/cocaine-tools|uhriap-actions.auth
⇒  cocaine-tool auth edit -necho
Service storage has been disconnected
```

**Pull request merged**